bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_GluGluH_01J_HToAATo4B_M-MASS/SUSY_GluGluH_01J_HToAATo4B_M-MASS_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model NMSSMHET --modelname
+
+generate p p  > h01 $$ w+ w- z , (h01 > a01 a01, a01 > b b~, a01 > b b~)
+add process p p  > h01 j $$ w+ w- z , (h01 > a01 a01, a01 > b b~, a01 > b b~)
```

Comment:
What does $$ w+ w- z do here?

Reply:
Hi @sihyunjeon , it's intended to exclude VH associated production with hadronic W/Z decays.  Perhaps this is not necessary since I only specify at most 1 jet?

Reply:
Exactly. Just to let you know, MG first scans QCD possibilities so even if you had 2 jets, "j > j j" so "W/Z > j j" won't be present anyways if you do not add additional commands in your lines such as QED=2

Reply:
OK, removed now.